image.yaml

```diff
@@ -46,10 +46,10 @@ modules:
             version: '1.0'
 
 artifacts:
-      - md5: d407ad9f1e605f2a4cec16ffbea7b3ce
-        name: jboss-eap-7.2.6-patch
-        target: jboss-eap-7.2.6-patch.zip
-        url: http://$DOWNLOAD_SERVER/devel/candidates/JBEAP/JBEAP-7.2.6.CP-CR1/jboss-eap-7.2.6.CP-CR1-patch.zip
+      - md5: 34a8787d5f5cdf54d5c1477ab1bdf47b
+        name: jboss-eap-7.2.7-patch
+        target: jboss-eap-7.2.7-patch.zip
+        url: http://$DOWNLOAD_SERVER/devel/candidates/JBEAP/JBEAP-7.2.7.CP-CR3/jboss-eap-7.2.7.CP-CR3-patch.zip
```

Comment:
Also, note you'll need to fetch:

https://github.com/jboss-container-images/jboss-eap-7-image/blob/7.2.x/modules/7.2.7/module.yaml#L20

Reply:
@luck3y Is the edit correct?

Reply:
@drichtarik LGTM!

Reply:
@drichtarik @hmlnarik @luck3y All seems to be OK now. Thanks to all!